lib/framedivider.rb

```diff
@@ -1,22 +1,22 @@
 require "openstudio"
 
-
 ##
 # Flatten OpenStudio 3D points vs Z-axis (Z=0).
 #
-# @param [Array] pts Point3D array
+# @param [Array] pts OpenStudio Point3D array/vector
 #
 # @return [Array] Flattened OS 3D points array
 def flatZ(pts)
-  raise "Invalid pts (flatZ)" unless pts
-
-  cl = OpenStudio::Point3dVector
-  valid = pts.is_a?(cl) || pts.is_a?(Array)
-  raise "#{pts.class}? expected #{cl} (flatZ)" unless valid
+  unless pts && (pts.is_a?(OpenStudio::Point3dVector) || pts.is_a?(Array))
+    TBD.log(TBD::DEBUG, "Invalid pts - flatZ")
```

Comment:
Is this really just a debug message?  Something has gone very wrong if you are seeing this right?

Reply:
See [here](https://github.com/rd2/tbd/pull/63#discussion_r665328560). I wasn't planning on a _production debugging_ solution - more of a basic development-only usage.